### ISSUE TITLE
Processing on dGPU even disabled it on settings

### ISSUE THREAD

**Issue by rafa9101** [Mastermindzh/tidal-hifi#521]
Environment:

Operating System: CachyOS
Desktop Environment: KDE Plasma 6.2.4
Kernel Version: Linux 6.12.6-2-cachyos
Graphics Platform: Wayland
Processors: 12th Gen Intel(R) Core(TM) i5-12500H (16) @ 4.50 GHz
Memory: 32GiB RAM
Graphics: 
   Intel Iris Xe Graphics @ 1.30 GHz [Integrated]
   NVIDIA GeForce RTX 4060 Max-Q / Mobile [Discrete]

tidal-hifi Version: 5.17.0 (AUR)

Issue Description:
Opening Tidal-HiFi, having disabled GPU processing toggle on the app settings, I can see a process of Tidal running as gpu-process, using the dGPU even on battery, this keeps the dGPU powered on until I close Tidal, draining battery unnecesary.

**Comment by Mastermindzh** [comment]
Hey,

I'd have to dive into this.
Our app doesn't control the `type=` flag but it **shouldn't** be passing the `--enable-gpu-rasterization` flag.

Did you modify the AUR instance in any way?
What is logged when you start tidal from the CLI? (you should **not** see logs like these: `enabling command line option ${flag}` for the values you haven't enabled in settings.

**Comment by rafa9101** [comment]
Hi!, sorry for delayed response, and not, didn't shows this logs 

![Image](https://github.com/user-attachments/assets/aa01ae2c-53a0-43f4-9118-4886788955c7)

**Comment by Mastermindzh** [comment]
Yeah, that confirms that it isn't the app itself that is setting those flags. Very weird.

**Comment by Mastermindzh** [comment]
Since it isn't the app doing this but some external issue I will close it
